libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts

```diff
@@ -158,9 +163,51 @@ export class GenericDrivingLicenseApi
     return licenses
   }
 
+  private async getCachedGetPkPassToken(): Promise<string | null> {
+    if (!this.cacheManager) {
+      return null
+    }
+
+    const cached = await this.cacheManager?.get(CACHE_KEY)
+
+    if (cached && typeof cached === 'string' && cached.length > 0) {
+      return cached as string
+    }
+
+    return null
+  }
+
+  private parseTtlFromTokenExpiry(expiry?: string | null): number | null {
+    if (!expiry) {
+      return null
+    }
+
+    try {
+      const parsed: number =
+        Date.parse(expiry) - Date.now() - CACHE_TOKEN_EXPIRY_DELTA_IN_MS
```

Comment:
```suggestion
        Date.parse(expiry).getTime() - Date.now() - CACHE_TOKEN_EXPIRY_DELTA_IN_MS
```

I prefer explicit `.getTime()` calls, over implicit `.valueOf()`

Reply:
my bad. Date.parse() returns a number.